qubes-rpc/nautilus/qvm_copy_nautilus.py

```diff
@@ -31,4 +35,4 @@ def on_menu_item_clicked(self, menu, files):
                # Check if file is not gone
                if not file_obj.is_gone()]
         cmd.insert(0, '/usr/lib/qubes/qvm-copy-to-vm.gnome')
-        subprocess.call(cmd)
+        subprocess.Popen(cmd, shell=False)
```

Comment:
This leaves zombie processes. I see you want to prevent nautilus being unresponsive, but something needs to collect exit code from a child process to not leave zombies...

Reply:
Is it acceptable to set `SIG_IGN` for a child process?

Reply:
I'm afraid it may confuse nautilus in other places